doc/source/parameters/cfd/physical_properties.rst

```diff
@@ -0,0 +1,57 @@
+Physical Properties
```

Comment:
Should we add the non newtonian section here now or should we wait for it to be included in the multiple-fluid hierarchy?

Reply:
Definitely worth adding here now. We can add a comment that it is presently only supported for a single fluid but that upcoming support for multifluid is coming.
You want to take a jab at it  @CaroleAnneDaunais ?

Reply:
Yes of course!